src/core/AdminGuesser.tsx

```diff
@@ -33,7 +39,7 @@ const AdminGuesser = ({
   basename,
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   error = DefaultError as any,
-  store,
+  store = defaultStore,
```

Comment:
React-admin defines the default store in `react-adminù 's `Admin` component. Since `AdminGuesser` relies on the lower-level `AdminContext`, we have to repeat the same initialization.